### ISSUE TITLE
Right Click - Copy minimizes window to upper left corner of screen

### ISSUE THREAD

**Issue by giz83** [notepad-plus-plus/notepad-plus-plus#8103]
### Description of the Issue
On right-click & pick any command, Notepad++ window reduces to a minimal size in upper left-hand corner of display. It is NOT minimizing to taskbar.
Issue does not appear when Notepad++ Window is maximized.
Copy & paste with keyboard shortcuts are ok. (ctrl-c & ctrl-v)

### Steps to Reproduce the Issue
Right-click in document.
Select and command from context menu that appears.

### Expected Behavior
NOT minimize window.

### Actual Behavior
Window minimizes to upper left-hand corner of display

### Debug Information
Running Notepad++ version 7.8.5 32-bit
Notepad++ v7.8.5   (32-bit)
Build time : Mar  4 2020 - 11:04:20
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 1909
OS Build : 18363.720
Plugins : mimeTools.dll NppConverter.dll NppExport.dll 
Running Notepad++ version 7.8.5 32-bit
Notepad++ v7.8.5   (32-bit)
Build time : Mar  4 2020 - 11:04:20
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 1909
OS Build : 18363.720
Plugins : mimeTools.dll NppConverter.dll NppExport.dll 
 - Mime Tools v2.5
 - Npp Converter v4.2.1
 - NppExport v0.2.9

~ G

**Comment by giz83** [comment]
Running Notepad++ using "Runas Administrator" and the problem does not seem to be occuring.

**Comment by dail8859** [comment]
Similar to #4753?

**Comment by giz83** [comment]
Yes, similar. - running as administrator or avoiding right-clicking menu options is a work around.
NVIDIA NVS 4200M (Bios v75.19.53.0.22)
Driver Provider: NVIDOA
Driver Date: 5/1/2017
Driver Version: 22.21.13.8205
Digital Signer: Microsoft Windows Hardware Compatibility Publisher

nView Manager is installed, v6.14.10.14633. Both the nView and graphics drivers latest installed.

Thanks

**Comment by alankilborn** [comment]
@giz83 Still a problem with a newer (e.g. 8.4.4) version of Notepad++?

**Comment by D-Scaduto** [comment]
This is still a problem in the current latest version 8.4.7

**Comment by dalers** [comment]
Encountering same issue with 8.4.8 (64 bit) on Win10 executing NP++ as a normal user. 

I found a post from 2015 saying deleting the .ini file corrected behavior, but I can't find a .ini file (perhaps it's not used anymore?) https://community.notepad-plus-plus.org/topic/108/np-minimizing-on-cut-paste-search-and-most-other-actions

I tried deleting the cofiguration settings folder (AppData/Roaming/Notepad++/) per the NP++ manual, but it had no effect.  https://npp-user-manual.org/docs/config-files/

The Notepad++ config folder was recreated, but then re-starting NP++, opening a file, selecting text, and right-click/copy immediately minimised NP++ to the upper left monitor in a 2-monitor setup. However, using Ctrl-A to copy does cause minimization, so I just have to remember.... :-)

**Comment by shulist** [comment]
Four years later his bug is still evident in v8.6.  Is anybody looking at it? Does anybody care?

**Comment by alankilborn** [comment]
> Is anybody looking at it? Does anybody care?

Probably nobody that can fix it can every replicate it.
